lib/config.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+var Promise = require("bluebird");
+var Confidence = require("confidence");
+var Joi = require("joi");
+
+var configSettings;
+
+function Config() {
+	configSettings = {
+		bandwidth : {
```

Comment:
What do you think about externalizing this config as a JSON file? Then the `Config` constructor could take a path as its parameter, load the file, and instantiate the internal (private) config store.